### ISSUE TITLE
Referencing thread worker class variables from another thread

### ISSUE THREAD

**Issue by jondanb** [jondanb/classic-windows-task-manager-linux#3]
Referencing thread worker class variables from another thread is bad practice and shouldn't be done. Specifically in the class `CWTM_GlobalUpdateIntervalHandler`, the timeout interval should be re-assigned using signals and slots

```py
def switch_selected_tab_update_speed(self, thread_worker, update_interval):
        thread_worker.timeout_interval = update_interval
```

**Comment by jondanb** [comment]
Implemented `CWTM_TimeoutIntervalChangeSignal` in `qt_components.py`

**Comment by jondanb** [comment]
Problem in `CWTM_NetworkingInterfaceRetrievalWorker` in `get_networking_interface_usage_frame` in `thread_workers.py`
```py
self.parent.net_t_network_list_table.setRowCount(0)
```
Need to find a workaround in the main thread to set the row count to 0 before updating and populating the network interface information table

**Comment by jondanb** [comment]
`parent_tab_widget` variable in certain thread worker classes reference a variable outside of its own thread.

Possible way to fix: add the current tab index as an argument for the function and then connect the information frame function inside of a lambda function to the signal

**Comment by jondanb** [comment]
> Referencing thread worker class variables from another thread is bad practice and shouldn't be done. Specifically in the class `CWTM_GlobalUpdateIntervalHandler`, the timeout interval should be re-assigned using signals and slots
> 
> ```python
> def switch_selected_tab_update_speed(self, thread_worker, update_interval):
>         thread_worker.timeout_interval = update_interval
> ```

Fixed this in: 2ab1c68bee3895254f90a54921499e04f8f318e8
